### PR TITLE
[MORE] Create more command

### PR DESCRIPTION
**Pull request by seonggwonyoon**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature
**What is the current behavior? (You can also link to an open issue here)**
Not found `more` command
**What is the new behavior (if this is a feature change)?**
Create `more` command
**Additional info:**

### CODE REVIEW

**Comment by QuiiBz** [comment]
Hi! Looks like more allows few [options](https://man7.org/linux/man-pages/man1/more.1.html#OPTIONS)

**Comment by seonggwonyoon** [comment]
> 
> 
> Hi! Looks like more allows few [options](https://man7.org/linux/man-pages/man1/more.1.html#OPTIONS)

Hello! Added more options.

File: dev/more.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export const completionSpec: Fig.Spec = {
+  name: "more",
+  description: "opposite of less",
+  options: [
+    {
+      name: ["-d, --slient"],
+      description:
```
Comment (QuiiBz) [inline dev/more.ts:7]: Just a small typo:
```suggestion
    {
      name: ["-d, --silent"],
      description:
```

Comment (seonggwonyoon) [reply dev/more.ts]: Fixed it. Thanks @QuiiBz

**Review by QuiiBz — APPROVED** [review]
Cool addition 👍

**Pull request merged**